src/govuk/components/radios/radios.yaml

```diff
@@ -601,3 +602,190 @@ examples:
+      - value: no
+        text: No
+- name: attributes
+  hidden: true
+  data:
+    name: example-name
+    attributes:
+      data-attribute: value
+      data-second-attribute: second-value
+    items:
+      - value: yes
+        text: Yes
+      - value: no
+        text: No
+- name: items with attributes
+  hidden: true
+  data:
+    name: example-name
+    items:
+      - value: yes
+        text: Yes
+        attributes:
+          data-attribute: ABC
+          data-second-attribute: DEF
+      - value: no
+        text: No
+        attributes:
+          data-attribute: GHI
+          data-second-attribute: JKL
+- name: with empty conditional
+  hidden: true
+  data:
+    name: example-conditional
+    items:
+      - value: yes
+        text: Yes
+        conditional:
+          html: false
+      - value: no
+        text: No
+- name: label with classes
+  hidden: true
+  data:
+    name: example-label-classes
+    items:
+      - value: yes
+        text: Yes
+        label:
+          classes: bold
+- name: with hints on parent and items
+  hidden: true
+  data:
+    name: example-multiple-hints
+    fieldset:
+      legend:
+        text: Have you changed your name?
+    hint:
+      text: This includes changing your last name or spelling your name differently.
+    items:
+      - hint:
+          text: Hint for yes option here
+      - hint:
+          text: Hint for no option here
+- name: with describedBy and hint
+  hidden: true
+  data:
+    name: example-hint-describedby
+    fieldset:
+      describedBy: some-id
+      legend:
+        text: Have you changed your name?
+    hint:
+      text: This includes changing your last name or spelling your name differently.
+- name: with error message
+  hidden: true
+  data:
+    name: example-error-message
+    errorMessage:
+      text: Please select an option
+- name: with error message and idPrefix
+  hidden: true
+  data:
+    name: example-error-message
+    idPrefix: id-prefix
+    errorMessage:
+      text: Please select an option
+- name: with hint and error message
+  hidden: true
+  data:
+    name: example-error-message-hint
+    errorMessage:
+      text: Please select an option
+    fieldset:
+      legend:
+        text: Have you changed your name?
+    hint:
+      text: This includes changing your last name or spelling your name differently.
+- name: with hint, error message and describedBy
+  hidden: true
+  data:
+    name: example-error-message-hint
+    errorMessage:
+      text: Please select an option
+    fieldset:
+      describedBy: some-id
+      legend:
+        text: Have you changed your name?
+    hint:
+      text: This includes changing your last name or spelling your name differently.
+- name: label with attributes
+  hidden: true
+  data:
+    items:
+      - value: yes
+        text: Yes
+        label:
+          attributes:
+            data-attribute: value
+            data-second-attribute: second-value
+- name: fieldset params
```

Comment:
Just a small thing but I **think** this example should be called 'fieldset with attributes' to be consistent with the the other examples.

Reply:
@hannalaakso I think this test was about testing whether params are passed correctly to the fieldset component - `classes`, `legend` and anything related to having a hint or error message on the input, as well as passing through attributes. I think I changed it from `fieldset with attributes` to `fieldset params` to keep it a bit more generic to try and reflect this. But that may not be perfect, I'm open to suggestions!

Reply:
That makes sense. Could maybe call it `fieldset params for hint and error message` but that's just a suggestion.